src/apps/ensuro/polygon/ensuro.etoken.token-fetcher.ts

```diff
@@ -0,0 +1,56 @@
+import { Inject } from '@nestjs/common';
+
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+import { AppTokenTemplatePositionFetcher } from '~position/template/app-token.template.position-fetcher';
+import {
+  GetAddressesParams,
+  DefaultAppTokenDefinition,
+  GetUnderlyingTokensParams,
+  UnderlyingTokenDefinition,
+  GetPricePerShareParams,
+  DefaultAppTokenDataProps,
+} from '~position/template/app-token.template.types';
+
+import { EnsuroContractFactory, EnsuroEtoken } from '../contracts';
+import { EnsuroApiRegistry } from '../common/ensuro.api-registry';
+
+const USDC = "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174";
+
+@PositionTemplate()
+export class PolygonEnsuroETokenTokenFetcher extends AppTokenTemplatePositionFetcher<EnsuroEtoken> {
+  groupLabel = "eTokens";
+
+  constructor(
+    @Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit,
```

Comment:
For CI to be 🟢  you will need to change the access modifier to `protected` because the base class implemented app toolkit `protected`

Reply:
Done, I just pushed the commit